installation-bundle/src/Database/Version330Update.php

```diff
@@ -12,17 +12,35 @@
 
 namespace Contao\InstallationBundle\Database;
 
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
 use Contao\StringUtil;
+use Doctrine\DBAL\Connection;
 
 /**
  * @internal
  */
-class Version330Update extends AbstractVersionUpdate
+class Version330Update extends AbstractMigration
 {
+    /**
+     * @var Connection
+     */
+    protected $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function getName(): string
+    {
+        return 'Contao 3.3.0 Update';
+    }
+
     /**
      * {@inheritdoc}
      */
-    public function shouldBeRun(): bool
+    public function shouldRun(): bool
```

Comment:
can we keep a method alias and forward the call? I currently use these classes to execute Contao updates in doctrine migrations.

Reply:
Agree, will fix.
*(but your migrations probably will still need to be updated because of other changes to these classes)*

Reply:
_I wasn't talking about **my** migrations, but about executing the Contao core migrations from Doctrine migrations. The constructor arguments are the same, so I don't see any changes 🙃_

Reply:
With *“Your migrations”* I was referring to your doctrine migrations 🙃
But you are right, I forgot that I kept the constructor the same.